Client/Frontend/Home/JumpBackIn/Cell/SyncedTabCell.swift

```diff
@@ -346,3 +356,51 @@ extension SyncedTabCell: Notifiable {
         }
     }
 }
+
+// MARK: - Contextual Hints related
+extension SyncedTabCell: FeatureFlaggable {
+    typealias CFRPrefsKeys = PrefsKeys.ContextualHints
+
+    private func prepareSyncedTabOnJumpBackInContextualHint(with profile: Profile) {
+        guard contextualHintViewController.shouldPresentHint(),
+              featureFlags.isFeatureEnabled(.contextualHintForJumpBackInSyncedTab, checking: .buildOnly)
+        else {
+            profile.prefs.setBool(false, forKey: CFRPrefsKeys.jumpBackInSyncedTabConfiguredKey.rawValue)
+            return
+        }
+
+        contextualHintViewController.configure(
+            anchor: cardTitle,
+            withArrowDirection: .down,
+            andDelegate: BrowserViewController.foregroundBVC(),
+            presentedUsing: presentContextualHint,
+            withActionBeforeAppearing: prepareToPresentHint,
+            actionOnDismiss: nil,
+            andActionForButton: nil,
+            andShouldStartTimerRightAway: true) {
+                profile.prefs.setBool(true, forKey: CFRPrefsKeys.jumpBackInSyncedTabConfiguredKey.rawValue)
+            }
+    }
+
+    private func presentContextualHint() {
+        let bvc = BrowserViewController.foregroundBVC()
```

Comment:
Are we not passing this in with DIP somehow? I forget. I thought that was the purpose of that.

Reply:
We spoke about this and it can be ifnored for now.